### PR TITLE
Add markdeep

### PR DESCRIPTION
**Pull request by Piicksarn**
@cdnjs/intern @cdnjs/intern2,
Please help me for checking the PR, which is for #9052, thank u!

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.
# Profile of the lib
- Git repository (required): https://github.com/morgan3d/markdeep
- Official website (optional, not the repository): https://casual-effects.com/markdeep/
- NPM package url (optional):
- GitHub / Bitbucket popularity (required):
  - Count of stars: 14
  - Count of watchers: 3
  - Count of forks: 0
- NPM download stats (optional):
  - Downloads in the last day:
  - Downloads in the last week:
  - Downloads in the last month:
# Essential checklist
- [ ] I'm the author of this library
  - [ ] I would like to add link to the page of this library on CDNJS on website / readme
- [x] This lib was not found on cdnjs repo
- [x] No already exist / duplicated issue and PR
- [ ] The lib has notable popularity
  - [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  - [ ] More than 500 downloads stats per month on npm registry
- [x] Project has public repository on famous online hosting platform (or been hosted on npm)
# Auto-update checklist
- [ ] Has valid tags for each versions (for git auto-update)
- [ ] Auto-update setup
- [ ] Auto-update target/source is valid.
- [ ] Auto-update filemap is correct.
# Git commit checklist
- [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
- [x] The parent of the commit(s) in the PR is not old than 3 days.
- [x] Pull request is sending from a non-master branch with meaningful name.
- [x] Separate unrelated changes into different commits.
- [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
- [x] Close corresponding issue in commit message
- [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by pvnr0082t** [comment]
LGTM

File: ajax/libs/markdeep/0.1.0/markdeep.min.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
+/**
+  markdeep.js
+  Version 0.01
```
Comment (maruilian11) [inline ajax/libs/markdeep/0.1.0/markdeep.min.js]: why change the version number named by the author?

**Comment by maruilian11** [comment]
@PeterDaveHello in this case, we can't set up the auto-update config for this lib, do we need to open an issue to ask the author if he can use git tags in later versions?

File: ajax/libs/markdeep/0.1.0/markdeep.min.js (hunk 1 above)
Comment (Piicksarn) [inline ajax/libs/markdeep/0.1.0/markdeep.min.js]: For the consistency of most version representation of cdnjs.

**Comment by PeterDaveHello** [comment]
You shout not touch the version, are you the author? I don't think so.

**Comment by PeterDaveHello** [comment]
@Piicksarn why don't you mention the auto-update and the touched version change? You should contact with the author for the possibility of auto-update.

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by PeterDaveHello** [comment]
Close due to no response.

**Pull request closed without merging**